src/assets/scripts/client/ui/TrafficRateController.js

```diff
@@ -226,6 +251,34 @@ export default class TrafficRateController {
         return $element;
     }
 
+    /**
+     * Build form element
+     *
+     * @for TrafficRateController
+     * @method _buildWindSlider
+     * @param key {string}
+     * @param value {number} passed to the change handler
+     * @param onChangeMethod {function}
+     * @param min {int} min value of slider
+     * @param max {int} max value of slider
+     * @param step {int} step value of slider
+     * @return {jquery|HTML Element}
+     */
+    _buildWindSlider(key, label, value, onChangeMethod, min, max, step) {
+        const template = `
+            <div class="form-element">
```

Comment:
take a look at the `SELECTORS` constant we have setup.  for something like this, where will be accessing this later via js, we should add an additional class prefixed with `js`.  in this case maybe something like `js-windSlider` or something similar.

The reason for that is `.form-element` could be used on other forms that might be in the DOM.  using a specific `js` prefixed classname makes it obvious that A) it isn't something that is styled, B) it will be used by js at some point and C) it's unique

Reply:
@n8rzz I am happy to make these changes, but I just want to point out that I was following the pattern used nearby in the same file to create the traffic rate sliders. Should I correct those as well?

Reply:
see above